nulib/meadow
# Switch from webpack to esbuild

# Summary 

Switch Meadow front end from webpack to esbuild

# Specific Changes in this PR
- Configure esbuild for `npm run deploy` and `npm run watch`
- Fix issue in `elasticsearch.js` that was preventing it from building under esbuild
- Configure Meadow to use esbuild as its file watcher

- list changes
# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

- Make sure the entire front end works in dev
- We won't be able to test the packaged build until it's on staging

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
